### ISSUE TITLE
Commenting the build scan URL does not seem to work for contributor PRs

### ISSUE THREAD

**Issue by britter** [gradlex-org/build-parameters#85]
See [this build](https://github.com/gradlex-org/build-parameters/actions/runs/4098548039/jobs/7067737228) that was started for #84. First the Gradle build step failed but then also the commenting of the build scan URL failed. Maybe it's related to permissions? Maybe we should just drop that part...

**Comment by SimonMarquis** [comment]
I think you probably need to ask for the corresponding permission

```
permissions:
  pull-requests: write
```

**Comment by SimonMarquis** [comment]
I've tested this hypothesis in https://github.com/gradlex-org/build-parameters/pull/86 but it does not seem to work either. It might be a repo configuration thing.

**Comment by SimonMarquis** [comment]
It turns out since ~2020, changes to GitHub Actions permissions made commenting on a PRs coming from forks impossible with the `pull_request` event.

TL;DR
> Any automated processing of PRs from an external fork is potentially dangerous and such PRs should be treated like untrusted input.
> https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

**Comment by britter** [comment]
Thank you for getting to the bottom of this. I guess in that case I will just remove this because the build scan link is available from the summary of the job run overview page.
